src/cli/cli.hpp

```diff
@@ -661,6 +664,9 @@ class Interpreter : public Output
     // Commands supported by radio:
     // [diag, help, reset, version]
     static constexpr Command sCommands[] = {
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE
+        {"allowrouterid", &Interpreter::ProcessAllowRouterId},
```

Comment:
nit: how about rename to `routeridrange`?
```suggestion
        {"routeridrange", &Interpreter::ProcessAllowRouterId},
```

If you agree, also update the method to `ProcessRouterIdRange()`.